server/graphql/v2/object/Host.js

```diff
@@ -355,6 +356,7 @@ export const Host = new GraphQLObjectType({
           limit: { type: new GraphQLNonNull(GraphQLInt), defaultValue: 100 },
           offset: { type: new GraphQLNonNull(GraphQLInt), defaultValue: 0 },
           state: { type: GraphQLString, defaultValue: null },
+          orderBy: { type: OrderByInput, defaultValue: { field: 'createdAt', direction: 'DESC' } },
```

Comment:
Anything wrong with `ChronologicalOrderInput` which was suggested?

Reply:
`ChronologicalOrderInput` is acting weird. There are default value in the definition of the input but when I do `orderBy: {type : ChronologicalOrderInput}`, I have an error saying `cannot read field attribute from undefined`.
In `pendingApplications` query field who is using `ChronologicalOrderInput`, `defaultValue: { field: 'createdAt', direction: 'DESC' }` is specified. If I do this it works but it seems redondant with the definition of `ChronologicalOrderInput`.
Furthermore, do we really want a specific orderBy for date instead of a generic one ?
What do you prefer ?

Reply:
In most cases, I see that it's used like the following:

```
    orderBy: {
      type: new GraphQLNonNull(ChronologicalOrderInput),
      description: 'The order of results',
      defaultValue: CHRONOLOGICAL_ORDER_INPUT_DEFAULT_VALUE,
    },
```

Have you tried that way?

> Furthermore, do we really want a specific orderBy for date instead of a generic one ?

It's an established practice to use this specialized input for such case. Yes, I think it's good to use it there.